gitops/tekton-pipeline/patched/300-condition.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 The Tekton Authors
```

Comment:
is any file in gitops/tekton-pipeline/patched/ used?

Reply:
Folder will be removed in the next PR update.